lib/meilisearch-rails.rb

```diff
@@ -199,7 +202,13 @@ def to_settings
         settings = {}
         OPTIONS.each do |k|
           v = get_setting(k)
-          settings[k] = v unless v.nil?
```

Comment:
Hi @yagince! I'm sorry if I was not clear in my previous comment, but you don't need to handle the camelCase snake_case on the rails side. We have a handler in the meilisearch-ruby already.

So every attribute sent as snake_case or camelCase will be transformed accordingly. 
I meant that we need to teach our meilisearch-rails users to use the correct way, "snake_case." 

So, I think you can rollback this part of the commit :)

Reply:
By the way, it should work if you send `max_total_hits` instead of `maxTotalHits`...

Reply:
If max_total_hits is sent as is, the following error will occur.
```
     MeiliSearch::ApiError:
       400 Bad Request - Json deserialize error: unknown field `max_total_hits`, expected `maxTotalHits` at line 1 column 31. See https://docs.meilisearch.com/errors#bad_request.
```

so, I convert it here. how is it?

Reply:
https://github.com/meilisearch/meilisearch-ruby/blob/05f7f011a0dd8447eb38cd5f92d182c7ce2bf1f6/lib/meilisearch/utils.rb#L18-L24
It looks like you are converting here, but this process appears to be converting only the SETTINGS key.

```{ hoge_foo: { bar_baz: 100 } }```
↓
```{ hogeFoo: { bar_baz: 100 }```

If value is Hash, wouldn't that key be converted?

Reply:
Oh, ok, then, we should fix it accordingly on the ruby side.

Reply:
Understood! Thank you.